Makefile

```diff
@@ -44,7 +44,7 @@ INSTALL_RPMS_FROM_URL ?=
 #         - uncomment source build section, comment binary section
 #         - adjust binary version path - version variable is not passed to WORKSPACE file!
 OV_SOURCE_BRANCH ?= 2020.4
```

Comment:
Can you also change it to 2021.1?

Reply:
OV building from sources was not tested recently. It would require a few changes to enable it back. For now it is low priority.